### PR TITLE
Refactor file structure, file types, fix navbar and add privateroute …

### PR DESCRIPTION
**Pull request by leogoldberg**
refactor component files to jsx
refactor folder structure: components, util, style folders under src
move navbar component to app.js
add some basic auth methods in utils, using a dummy jwt for now
create privateroute component to handle redirect when user not logged in

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// TODO: edit to use real back end token instead of test key (TestLogin)
+const TOKEN_KEY = 'jwt';
```
Comment (jorawarc) [inline src/utils/index.js:2]: This is bundled in with [PR#2](https://github.com/c9y7ip/ezpark-admin/pull/2/files)

**Comment by jorawarc** [comment]
I think we are going to have to discuss how we want to structure the admin side before merging this, mainly because I am unsure how this change will integrate with with [PR#2](https://github.com/c9y7ip/ezpark-admin/pull/2/files)

File: src/components/Login.jsx [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-    onEmailChange(e) {
-        this.setState({email: e.target.value});
-    }
 
     handleSubmit(event){
         event.preventDefault();
-        EventService.auth.userLogin(this.state, (authSuccess) => {
+
+        const email = Document.getElementById("email").value
+        const password = Document.getElementById("password").value
```
Comment (JasonXD-CS) [inline src/components/Login.jsx]: Are you sure this works in React? Also isn't it lowercase `document`

Comment (leogoldberg) [reply src/components/Login.jsx]: Yea that's true actually, it doesn't work. I will change it back to the onEventChange listeners

**Review by JasonXD-CS — APPROVED** [review]
LGTM

**Pull request merged**